inspirehep/base/searchext/mappings/hep.json

```diff
@@ -57,15 +57,32 @@
                         }
                     }
                 },
+                "abstracts": {
+                    "type": "object",
+                    "properties": {
+                        "value": {
+                            "type": "string",
+                            "copy_to": ["abstract", "global_fulltext"]
+                        }
```

Comment:
I think for `abstract` and `title` we should check how to better tokenize also considering LaTeX formulas. I am not sure yet how, though.